run/btcwallet.go

```diff
@@ -22,6 +23,8 @@ import (
 	"github.com/stroomnetwork/btcwallet/wallet"
 )
 
+const ethChangeAddr = "0x7b3f4f4b3cCf7f3fDf3f3f3f3f3f3f3f3f3f3f3f"
```

Comment:
this is a made up address that is used as a seed to generate btc address
if there is a collisions we can spend their money  :) it's kind of impossible to generate two same addresses even if you try really hard, otherwise you can't be sure that somebody generates this address in the future